test/start-operator.sh

```diff
@@ -18,7 +18,7 @@ echo "Deploying '${deploy}'..."
 
 if [ -f "$deploy" ]; then
   tmpdir=$(${SSH} mktemp -d)
-  trap '${SSH} "echo cleaning up temp folder: $tmpdir; rm -rf $tmpdir"' SIGTERM SIGINT EXIT
+  trap '${SSH} "rm -rf $tmpdir"' SIGTERM SIGINT EXIT
```

Comment:
@avalluri this debug output is no longer needed, right?

Reply:
Yes, it's ok to remove.